src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -265,8 +269,8 @@ def _sum_to_dynatrace_metric(self, metric: Metric, point: NumberDataPoint):
                 dict(point.attributes),
                 int(point.time_unix_nano / 1000000))
 
-    def _gauge_to_dynatrace_metric(self, metric: Metric,
-                                   point: NumberDataPoint):
+    def _non_monotonic_to_dynatrace_metric(self, metric: Metric,
```

Comment:
```suggestion
    def _non_monotonic_sum_to_dynatrace_metric(self, metric: Metric,
```

Reply:
Here, sum was left out intentionally, as we also use this for gauges.

Reply:
Maybe we should just call it `_to_dynatrace_gauge` then?

Reply:
Naming. :man_facepalming:  
[7e63c16](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/7e63c16965260103da8f48a1ad59c0afca470eb8)